executers/utils/utils.go

```diff
@@ -124,13 +86,13 @@ func GetRegex() (regExp *RegExp, err error) {
 	return
 }
 
-func logError(err error) {
+func LogError(err error) {
```

Comment:
I think this function is redundant. Do we really need the *Received an error:* prefix? The logging already includes the *[ERROR]* prefix.

Reply:
This function is needed not to replicate the if statement. I've removed the prefix